tests/python/unittest/test_executor.py

```diff
@@ -72,7 +72,9 @@ def check_bind_with_uniform(uf, gf, dim, sf=None, lshape=None, rshape=None):
     assert_almost_equal(rhs_grad.asnumpy(), rhs_grad2, rtol=1e-5, atol=1e-5)
 
 
-@with_seed(0)
+# @roywei: Removing fixed seed as flakiness in this test is fixed
+# tracked at: https://github.com/apache/incubator-mxnet/issues/11685
```

Comment:
Feel free to remove those comments and simply link the issue in description of PR.